src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -21,8 +32,77 @@ public class Drive extends SubsystemBase {
     private double forward;
     private double rotation;
 
+    ChassisSpeeds speed =
+            ChassisSpeeds.fromFieldRelativeSpeeds(vx, vy, Math.PI / 2.0, this.getGyroRotation2d());
+    Pose2d pose = Pose2d(vx, vy, omega);
+    DifferentialDriveKinematics kinematics =
+            new DifferentialDriveKinematics(Units.inchesToMeters(27.0));
+
     public Drive(DriveIO io) {
         this.io = io;
+        configurePathPlanner();
+    }
+
+    private Pose2d Pose2d(double vx2, double vy2, double omega2) {
+        // TODO Auto-generated method stub
+        throw new UnsupportedOperationException("Unimplemented method 'Pose2d'");
+    }
```

Comment:
Could we delete this since it only exists to throw this exception lol

Reply:
Somehow the Pose2d object cannot be created without this